.github/workflows/_e2e_tests.yaml

```diff
@@ -1,17 +1,28 @@
 name: e2e tests
-run-name: e2e tests, branch:${{ github.ref_name }}, triggered by @${{ github.actor }}
```

Comment:
I'm curious: any particular reason to get rid of this? Was that too verbose or didn't convey useful information?

Reply:
In the case of the reusable workflows the `run-name` and even `name` itself are not that relevant as they're not used (at least from my experience) anywhere. It's because the reusable workflows (ones that have only `on.workflow_call` defined) are not run on their own. They do not appear in the workflows view on GH. When they're called by other workflows, they are run as jobs and get their name from the job definition in the workflow, e.g.

```
  build-image:
    uses: ./.github/workflows/_docker_build.yaml
``` 

will have `build-image` name, not the one that's specified in the `_docker_build.yaml`.